fixit/common/config.py

```diff
@@ -95,35 +90,69 @@ def get_validated_settings(
         # Set path setting to absolute path.
         file_content[path_setting_name] = str(abspath)
 
+    file_content["inherit"] = file_content.get("inherit", False)
+    file_content["allow_list_rules"] = file_content.get("allow_list_rules", [])
+
     return file_content
 
 
 @lru_cache()
-def get_lint_config() -> LintConfig:
+def _get_config_schema() -> Dict[str, Any]:
+    # __package__ should never be none (config.py should not be run directly)
+    # But use .get() to make pyre happy
+    pkg = globals().get("__package__")
+    assert pkg, "No package was found, config types not validated."
+    config = pkg_resources.read_text(pkg, LINT_CONFIG_SCHEMA_NAME)
+    schema = json.loads(config)
+    return schema
+
+
+CACHE: Dict[Path, LintConfig] = {}
+
+
+def get_lint_config(path: Optional[Path] = None) -> LintConfig:
```

Comment:
any reason for manually implementing that `CACHE` instead of applying `lru_cache`?